src/main/io/beeper.c

```diff
@@ -55,13 +55,18 @@ PG_REGISTER_WITH_RESET_TEMPLATE(beeperDevConfig_t, beeperDevConfig, PG_BEEPER_DE
 #endif
 #ifdef BEEPER
 #define BEEPER_PIN      BEEPER
+#define BEEPER_PWM_FREQ 0
+#elif defined(BEEPER_PWM)
+#define BEEPER_PIN      BEEPER_PWM
```

Comment:
@mikeller: Beeper on single pin will break eventually. You can use (dshot) ESC or VTX to beep, maybe PWM output can be used ... In case of ESC, multiple BEEPER destinations make sense (beep all motors at once)